.env.example

```
@@ -3,5 +3,7 @@ NEWSDOM_AUTH_MODE=required
 NEWSDOM_RUNTIME_PROFILE=production
 # Generate a high-entropy token in the deployment secret store. Never commit it.
 NEWSDOM_API_TOKEN=
+# Per-process non-waiting parser admission budget. Scale replicas separately.
+NEWSDOM_MAX_CONCURRENT_PARSES=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEWSDOM_MAX_CONCURRENT_PARSES key should go before the NEWSDOM_RUNTIME_PROFILE key

(UnorderedKey)